fault/tester/abstract_tester.py

```diff
@@ -0,0 +1,231 @@
+import magma as m
+from abc import abstractmethod
+from ..wrapper import CircuitWrapper
+from ..select_path import SelectPath
+from ..wrapped_internal_port import WrappedVerilogInternalPort
+from ..magma_utils import is_recursive_type
+
+
+class AbstractTester:
```

Comment:
`AbstractTester` -> `TesterBase` (only because `AbstractTester` could sound like `SymbolicTester` which is an actual thing :D here)